README.md

```diff
@@ -14,17 +14,16 @@ impress.js may not help you if you have nothing interesting to say ;)
 HOW TO USE IT
 ---------------
 
-[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke ;)
+[Use the source](http://github.com/impress/impress.js/blob/master/index.html), Luke!
```

Comment:
@FagnerMartinsBrack @henrikingo Thanks for you input guys. I just commited an update to this section. Wanted to make a PR of it, but pushed to master directly thanks to GitHub UI ;)
Sorry for that. I hope it will makes things more clear about the docs and less offensive.

Feel free to update the text.